### PR TITLE
Added utility tokens

### PR DESCRIPTION
**Pull request by Pallavi-PH**

### CODE REVIEW

File: scss/_core.scss [hunk 1]
```diff
@@ -17,3 +17,4 @@
 
 //Utiliticore/es
 @import "core/tokens/_sizing.scss";
+@import "core/tokens/_disable.scss";
```
Comment (visualapps) [inline scss/_core.scss:20]: Do we need a separate files for every tokens. I think this can go in one page.

Comment (visualapps) [reply scss/_core.scss:20]: https://www.lightningdesignsystem.com/design-tokens/

Comment (Pallavi-PH) [reply scss/_core.scss:20]: Ya sure, will make the changes

Comment (Pallavi-PH) [reply scss/_core.scss:20]: We have separate CSS files for all token classes. 
Screenshot from 2020-05-11 16-18-10
I think we can combine all tokens in on MDX file and keep separate CSS files?

Comment (visualapps) [reply scss/_core.scss:20]: we can use `utilities.scs`s for small tokens like, cursor-pointer, disabled etc.. These small classes do not deserve to have separate files.

Comment (Pallavi-PH) [reply scss/_core.scss:20]: Ok, sure

**Pull request closed without merging**